rclpy/rclpy/node.py

```diff
@@ -1150,9 +1151,14 @@ def create_publisher(
 
         publisher_handle = Handle(publisher_capsule)
 
+        if event_callbacks:
+            event_callbacks_with_logger = copy.copy(event_callbacks)
```

Comment:
why is this needed?

Reply:
I don't want to modify the `PublisherEventCallbacks`/`SubscriptionEventCallbacks` objects that the user pass in, in case they want to reuse them.

Reply:
This change is no longer relevant.